src/browser/provider/utils/client-functions.js

```diff
@@ -13,7 +13,17 @@ function getWindowDimensionsInfo () {
         availableHeight: screen.availHeight
     };
 }
-/*eslint-disable no-undef*/
+
+function getPageId () {
+    var array  = new Uint16Array(1);
+    var crypto = window.crypto || window.msCrypto;
+
+    crypto.getRandomValues(array);
```

Comment:
The `getRandomValues` fn returns the same array. So looks like it can be rewritten as: 
```JS
return crypto.getRandomValues(array)[0].toString()
```
Of course, it's minor

Reply:
The code piece 
```js
   var crypto = window.crypto || window.msCrypto;

    crypto.getRandomValues(array);

   return array[0].toString();
```

looks more readable then

```js
var crypto = window.crypto || window.msCrypto;

return crypto.getRandomValues(array)[0].toString();
```